### ISSUE TITLE
Improve app accessibility

### ISSUE THREAD

**Issue by Paulj1989** [NHS-South-Central-and-West/CAIP#38]
### Problem
Accessibility should be a central consideration when creating anything, and this should probably be even more of an emphasis for any NHS organisation! I've given quite a bit of thought to accessibility around the plots I've created (though I'm sure they could be improved), but there's lots of other ways that the app itself could be more accessible.

### Proposed Solution
Add alt text to plots, tables (if possible?), and any other elements that could give meaningful feedback that users with vision issues would benefit from.

### Alternatives
I suspect there's a bunch of different ways of doing this, and I imagine alt text is just one tool in a wider toolbox. I need to spend more time thinking about this and come up with other ways to improve the app's accessibility too.

### Additional Context
Tools for improving accessibility of R outputs:

- [BrailleR](https://github.com/ajrgodfrey/BrailleR)

Resources for improving the accessibility of apps:

- [Alt Text in R: Plots, Reports, and Shiny](https://www.jumpingrivers.com/blog/accessibility-alt-text-in-r/)
- [Image Alt Tag Tips in HTML](https://accessibility.psu.edu/images/imageshtml/)
- [Accessibility in R Applications](https://www.jumpingrivers.com/blog/accessible-shiny-standards-wcag/)
